tests/jest-dom-matchers/toBeInTheDocument.test.ts

```diff
@@ -1,10 +1,10 @@
 import { createTab } from 'test-mule';
 
 test.skip('toBeInTheDocument', async () => {
-	const { screen, utils } = await createTab();
-	await utils.injectHTML(`<button>Hi</button>`);
-	const inDoc = await screen.getByText('Hi');
-	await expect(inDoc).toBeInTheDocument();
-	await inDoc.parentNode.removeChild(inDoc);
-	await expect(expect(inDoc).toBeInTheDocument()).rejects.toThrow('foo');
-});
\ No newline at end of file
+  const { screen, utils } = await createTab();
+  await utils.injectHTML(`<button>Hi</button>`);
+  const inDoc = await screen.getByText('Hi');
+  await expect(inDoc).toBeInTheDocument();
+  await inDoc.parentNode.removeChild(inDoc);
```

Comment:
This won't work since we don't have synchronous DOM access. You'll have to use [`.evaluate`](https://playwright.dev/docs/api/class-jshandle#jshandleevaluatepagefunction-arg):

```suggestion
  await inDoc.evaluate((el) => el.remove())
```